modules/ossm-observability-cli.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+//* service_mesh/v2x/ossm-observability.adoc
+
+
+[id="ossm-observability-cli_{context}"]
+= Accessing metrics and tracing data from the CLI
+
+Access the Jaeger, Prometheus, and Grafana consoles to view and manage your data. 
+
+.Procedure
+
+. Run the following command to switch to the control plane project. In this example, `istio-system` is the control plane project.
```

Comment:
I'm having difficulty putting my finger on explicit instructions, but the convention in the doc set is to end the instructional text with a colon when it introduces a command a code block.  Here's sort of an example from the guidelines:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

My request is that you revise the instructional text in these steps to match the guidelines.